.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,181 @@
+name: Python CI
+on:
+  pull_request_target:
+    paths:
+      - python/**
+      - packages/sdk-codegen*/**
```

Comment:
monorepo - python CI should only run for files changed in these dirs... i think?

Reply:
That's a good start, I think. We may need `packages/sdk-codegen-scripts/**` as well but probably not since `sdk-codegen` is where any changes to the actual generation syntax would appear.

Reply:
actually, the trailing `*/**` should capture `sdk-codegen` and `sdk-codegen-{utils,scripts}`

Reply:
Doh. You're right. I didn't notice that `*` before the `/` ;)